### PR TITLE
Assingment_4 & Assingment 13 completed

### PR DESCRIPTION
**Pull request by omerana14**

### CODE REVIEW

**Review by MKadeer17 — APPROVED** [review]
Add indentation in Assignment 13 All Files.

**Review by poonam-khode — APPROVED** [review]

File: Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q1.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+            <button type="submit" onclick="manualTime()" class="btn btn-primary">
+                Click Here And See Date & Time in Local Format
+            </button>
+        </div>
+    </div>
+
+    <script>
+        function manualTime() {
+            let otherDate = new Date("02-21-2012 03:12");
```
Comment (Snehal-113) [inline Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q1.html]: need to convert this date into local timestamp-use toLocaleString() function.check it's implementation

File: Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q2.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            var weekday = new Array(7);
+
+            weekday[0] = "SU";
+            weekday[1] = "MO";
+            weekday[2] = "TU";
+            weekday[3] = "WE";
+            weekday[4] = "TH";
+            weekday[5] = "FR";
+            weekday[6] = "SA";
+
```
Comment (Snehal-113) [inline Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q2.html]: or u can write weekday=["SU","MO","TU"......,"SA"];

File: Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q3.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            var x = document.getElementById('chk_date').value;
+
+            var m_names = new Array
+                ("January", "February", "March",
+                    "April", "May", "June", "July",
+                    "August", "September", "October",
+                    "November", "December");
+
+            var d = new Date(x);
+            var twoDaysAgo = d.getDate(x) - 2;  //change day here
```
Comment (Snehal-113) [inline Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q3.html]: two should not be hard coded, if the user enters 1 day before or 5days before it must the show the date, this logic is not working if i give 1/5/2020

**Review by Snehal-113 — CHANGES_REQUESTED** [review]

File: Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q1.html (hunk 1 above)
Comment (omerana14) [reply Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q1.html]: function manualTime() {
            var d = new Date("Feb 21,2021 03:12:00");
            var n = d.toLocaleString('en-US', { year: "numeric", day: "numeric", month: "short", hour: "numeric", minute: "numeric", hour12: true });
            window.alert("Local Time & Date " + n);
        }


I use this logic here and now code works fine

File: Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q2.html (hunk 2 above)
Comment (omerana14) [reply Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q2.html]: weekday = ["SU", "MO", "TU", "WE", "TH", "FR", "SA"];

Done, made changes in array.

File: Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q3.html (hunk 3 above)
Comment (omerana14) [reply Ome/JavaScript/Assignment 13/JavaScriptAssignment13_Q3.html]: function preDate() {
            var x = document.getElementById('chk_date').value;

            var date = new Date(x);
            var twoDaysAgo = date - 1000 * 60 * 60 * 24 * 2;   // current date's milliseconds - 1,000 ms * 60 s * 60 mins * 24 hrs * (# of days beyond one to go back)
            twoDaysAgo = new Date(twoDaysAgo);
            alert("Two day's ago date is : " + twoDaysAgo);
        }

I used here this logic, is it work?

**Review by Snehal-113 — APPROVED** [review]

**Pull request merged**